api/v1alpha4/condition_consts.go

```diff
@@ -201,3 +201,14 @@ const (
 	// from making any further remediations.
 	TooManyUnhealthyReason = "TooManyUnhealthy"
 )
+
+// Conditions and condition Reasons for  MachineDeployments
+
+const (
+	// MachineDeploymentAvailableCondition means the MachineDeployment is available, that is, at least the minimum available
+	// machines required (i.e. Spec.Replicas-MaxUnavailable when MachineDeploymentStrategyType = RollingUpdate) are up and running for at least minReadySeconds.
+	MachineDeploymentAvailableCondition ConditionType = "Available"
+
+	// NotMinimumMachinesAvailableReason reflects the fact that the required minimum number of machines for a machinedeployment are not available.
+	NotMinimumMachinesAvailableReason = "NotMinimumMachinesAvailable"
```

Comment:
What about `WaitingForAvailableMachines`; also, should we drop available given that it is repetition considering it applies to `MachineDeploymentAvailableCondition`
Another alternative is to replace with something more user oriented, given that I'm not really sure the nuances of what available means are clear to the end users

Reply:
seems like everyone has a different opinion on this :laughing: @fabriziopandini @enxebre I'm fine with what ever you guys agree on. Please let me know.

Reply:
I'm good either way, let's go with @fabriziopandini suggestions if that make sense for everyone else
`WaitingForMachinesReason = WaitingForMachines`

Reply:
Done.